lldb/scripts/dpu/dpu_commands.py

```diff
@@ -58,7 +57,7 @@ def get_dpu_from_command(command, debugger, target):
         "dpu", "(struct dpu_t *)" + str(addr))
 
 
-def get_region_id_and_rank_id(rank, target):
+def get_rank_id(rank, target):
```

Comment:
This is find and does exactly what was done before but adapt for the new driver.
But we might want to change it as now the rank_id exposed in the `struct dpu_rank_t` is the real rank_id. We just need to do something clean to detect backend that are not hardware.I think, I'll do it after you will have push that.